winter-collector-page.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
your main page, for best practice should be called index.php

Reply:
changed file name!